File: lista1_js/34.media_numeros.js
//34.lista1 - Leia 3 números, calcule e escreva a média dos números.

import {question} from 'readline-sync'

//Entrada
const num1 = Number(question('Informe o primeiro número: '))
const num2 = Number(question('Informe o segundo número: '))
const num3 = Number(question('Informe o terceiro número: '))

//Processamento
const media = (num1 + num2 + num3) / 3

//Saída
console.log(`\n>>> A média dos números ${num1}, ${num2} e ${num3} é >>> ${media.toFixed(1)}.`)

File: lista1_py/34.media_numeros.py
#34.lista1 - Leia 3 números, calcule e escreva a média dos números.

#Entrada
num1 = int(input('Informe o primeiro número: '))
num2 = int(input('Informe o segundo número: '))
num3 = int(input('Informe o terceiro número: '))

#Processamento
media = (num1 + num2 + num3) / 3

#Saída
print(f'\n>>> A média dos números {num1}, {num2} e {num3} é >>> {media:.1f}.')